test/integration/http2_upstream_integration_test.cc

```diff
@@ -409,7 +409,11 @@ TEST_P(Http2UpstreamIntegrationTest, UpstreamResetWithManyStreams) {
                                                                    {":scheme", "http"},
                                                                    {":authority", "host"}},
                                            *responses[i]));
-           codec_client->sendData(*encoders[i], 0, true);
+           if (i % 15 == 0) {
```

Comment:
Heh, it was supposed to be UpstreamConnectionCloseWithManyStreams - I was thinking of connection reset and the ambiguity didn't occur to me.

I was just tossing them in there rather than creating another large many requests test. if you prefer I can copy-paste-edit so they're separate but for now I'll add a comment and see what you think :-)